tests/test_atmos.py

```diff
@@ -624,3 +625,16 @@ def test_late_frost_days(self, atmosds):
         lfd = atmos.late_frost_days(tasmin, date_bounds=("04-01", "06-30"))
 
         np.testing.assert_allclose(lfd.isel(time=0), exp, rtol=1e-03)
+
+
+def test_chill_units(atmosds):
```

Comment:
I am not sure whether to put these tests into test_atmos or test_temperature. In the end, this is some kind of integration test since I am even using the new helper function to compute the hourly data.

Reply:
Both are correct (sadly :sweat:).